docs/experience/user-satisfaction.md

```diff
@@ -0,0 +1,67 @@
+title: User Satisfaction
+description: How user satisfaction is measured in Sematext Experience
+
+On the Overview page of your Experience App you will notice several scores that show the overall user satisfaction for the related performance section of your website.
+
+<img
+  class="content-modal-image"
+  alt="Sematext Experience Satisfaction Scores"
+  src="../../images/experience/satisfaction.png"
+  title="Satisfaction Scores"
+  width=470
+  height=130
+/>
+
+You can hover over each of these scores to get additional information such as the exact numeric score (0.00–1.00), number of events, and the score range of different score levels. This score is based on the [Apdex](http://www.apdex.org/) industry standard for measuring users satisfaction based on the response time.
+
+
+### How It's Measured
+
+Different websites have different performance goals which is why the first step in getting good results in measuring your website's performance is to define the different time thresholds based on your requirements. These time thresholds are the key component in measuring users' satisfaction as the Apdex score is based on a set of response time measurements against the time threshold.
+
+Page loads, ajax requests, and on-page transactions have different performance characteristics so we separate these into different user satisfaction scores and allow website owners to set different time thresholds for each. You can configure each of these time thresholds by opening your App and clicking on Configure Requirements in the left menu.
+
+<!-- Todo: add link to a more detailed page about configuring requirements -->
```

Comment:
@adnanrahic & @amir-hadzic should we create a ticket for this?

Reply:
Yep. @amir-hadzic you have context. Do you know exactly what the detailed page should contain?